src/dal/src/projects.js

```diff
@@ -33,10 +33,12 @@ const getProjects = async () => {
   return result.data || [];
 };
 
-exports.initProjects = async (pool, { insertMfNProject, insertProject , insertProjectInstitutions, insertInstitution }) => {
+exports.initProjects = async (pool, { insertMfNProject, insertProject , insertProjectInstitutions, insertInstitution, insertProjectsInfrastructure,
+insertProjectsCollections }) => {
   try {
     const projects = await getProjects(); // eslint-disable-line no-await-in-loop
-    //console.log(projects);
+    if (projects.length === 0) throw "MWC did not return projects data";
```

Comment:
That's nice! Why a test on #projects <50 though? Does the frontend get problems otherwise?